protocol/x/accountplus/keeper/timestampnonce.go

```diff
@@ -115,3 +120,12 @@ func isLargerThanSmallestValue(value uint64, values []uint64) (bool, int) {
 		return false, minIndex
 	}
 }
+
+func contains[T comparable](list []T, value T) bool {
```

Comment:
Make function public so we can write separate unit tests.

Also since if we made this generalized, might as well move to [here](https://github.com/dydxprotocol/v4-chain/blob/18b7707a932cf9b06552a88178770e717e688965/protocol/lib/collections.go#L9)